src/main/java/mate/academy/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Dao;
+import mate.academy.model.CinemaHall;
+import mate.academy.util.HibernateUtil;
+
+@Dao
+public class CinemaHallDaoImpl extends AbstractDao implements CinemaHallDao {
+    public CinemaHallDaoImpl() {
+        super(HibernateUtil.getSessionFactory());
+    }
```

Comment:
Checklist: Don't create a default constructor when it is not needed. Since the superclass constructor is being called with `HibernateUtil.getSessionFactory()` which seems to be a constant operation for DAO implementations, you might want to consider removing this default constructor if it's not being overridden for a specific reason.